syntax/arrai.wbnf

```diff
@@ -20,7 +20,7 @@ expr   -> C* amp="&"* @ C* arrow=(
         > C* @ postfix=/{count|single}? C* touch? C*
         > C* (get | @) tail_op=(safe_tail | tail)* C*
         > %!patternterms(expr)
-        | C* cond=("cond" "(" (key=@ ":" value=@):",",? ("*" ":" f=expr ","?)? ")") C*
+        | C* cond=("cond" "{" (key="(" @ ")" ":" value=@):",",? ("_" ":" f=expr ","?)? "}") C*
```

Comment:
Since we're going to relax the rule that all expressions in patterns must be enclosed in `(...)`, let's remove the requirement here too.